internal/security/proxy/service.go

```diff
@@ -96,23 +95,13 @@ func (s *Service) Init(cert CertificateLoader) error {
 		return err
 	}
 
-	services := config.ListDefaultServices()
-	//TODO: There are specific other services being routed through Kong. This should probably be driven through configuration
-	//      For now, I'm putting this inline
-	services["edgex-support-rulesengine"] = "Rulesengine"
-	services["edgex-device-virtual"] = "Virtualdevice"
-
-	for _, name := range services {
-		params := Configuration.Clients[name]
-		if params.Host == "" {
-			LoggingClient.Warn("missing config information for service: " + name)
-			continue
-		}
+	// EFC issue_1662
```

Comment:
what is the purpose of this comment? if this is just about the fact that we added this function for this issue, I don't think that information belongs in the code, I think a better place is in the git commit message

Reply:
@anonymouse64 - removed this as well.  I had this in there for my own analysis and debugging, testing.  Removed :)